tests/models/layoutlmv2/test_processor_layoutlmv2.py

```diff
@@ -193,6 +198,22 @@ def preprocess_data(examples):
 
         self.assertEqual(len(train_data["image"]), len(train_data["input_ids"]))
 
+    def test_model_specific_kwargs(self):
+        if "image_processor" not in self.processor_class.attributes:
+            self.skipTest(f"image_processor attribute not present in {self.processor_class}")
+        image_processor = self.get_component("image_processor", apply_ocr=True)
+        tokenizer = self.get_component("tokenizer", max_length=117, padding="max_length")
+
+        processor = self.processor_class(tokenizer=tokenizer, image_processor=image_processor)
+
+        image_input = self.prepare_image_inputs()
+        with self.assertRaises(ValueError):
```

Comment:
Can we add a comment for ourselves on why an error is expected here?

Reply:
I just added this as comment:

`The processor expects 'text' to be provided when 'apply_ocr' is set to False`

resolving